### PR TITLE
[BF] fix fs and bidsignore input

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -187,21 +187,20 @@ else if (params.bids || params.bids_config){
...
             params.participants_label ? '--participants_label ' + participant_cleaned : ""
-            fs_flag =\
-            params.fs ? '--fs ' + freesurfer_path : ""
 
             clean_flag = params.clean_bids ? '--clean ' : ''
 
             """
             scil_validate_bids.py $bids_folder tractoflow_bids_struct.json\
-                --readout $params.readout $participants_flag $clean_flag $fs_flag -v
+                --readout $params.readout $participants_flag $clean_flag ${!fs_folder.empty() ? "--fs $params.fs" : ""} -v
```
Comment (AlexVCaron) [inline main.nf]: If you want to make it cleaner, you could define fs_flag (fs_flag = params.fs ? "--fs $params.fs" : "") above in a variable like in the older version and use it in the script string.

File: main.nf [hunk 2]
```diff
@@ -187,21 +188,18 @@ else if (params.bids || params.bids_config){
...
-            params.participants_label ? '--participants_label ' + participant_cleaned : ""
-            fs_flag =\
-            params.fs ? '--fs ' + freesurfer_path : ""
-
             clean_flag = params.clean_bids ? '--clean ' : ''
 
             """
             scil_validate_bids.py $bids_folder tractoflow_bids_struct.json\
-                --readout $params.readout $participants_flag $clean_flag $fs_flag -v
+                --readout $params.readout $clean_flag ${!fs_folder.empty() ? "--fs $params.fs" : ""} ${!bidsignore.empty() ? "--bids_ignore $params.bidsignore" : ""} -v
```
Comment (AlexVCaron) [inline main.nf]: After thoughts, for readability, either setup the `fs_folder` and `bidsignore` like the clean_flag or split the command on multiple lines with `\` like 
```
scil_validate_bids.py $bids_folder tractoflow_bids_struct.json\
    --readout $params.readout $clean_flag\
    ${!fs_folder.empty() ? "--fs $params.fs" : ""}\
    ${!bidsignore.empty() ? "--bids_ignore $params.bidsignore" : ""}\
    -v

**Review by AlexVCaron — CHANGES_REQUESTED** [review]
Tested and approved, after a few minor changes. Also, I just tested with relative paths as inputs. It works really well, though it kinda unfortunate that the bind paths need a `nextflow.config` to be passed. I just seed the idea, a `--binds` or `--bind_paths` parameter could be interesting, used in both `docker.runOptions` and `singularity.runOptions`, since we can give it by command line.

File: main.nf (hunk 2 above)
Comment (arnaudbore) [reply main.nf]: Done :)

File: main.nf [hunk 3]
```diff
@@ -187,21 +188,21 @@ else if (params.bids || params.bids_config){
...
-            params.participants_label ? '--participants_label ' + participant_cleaned : ""
-            fs_flag =\
-            params.fs ? '--fs ' + freesurfer_path : ""
-
             clean_flag = params.clean_bids ? '--clean ' : ''
 
             """
             scil_validate_bids.py $bids_folder tractoflow_bids_struct.json\
-                --readout $params.readout $participants_flag $clean_flag $fs_flag -v
+                --readout $params.readout $clean_flag\
```
Comment (AlexVCaron) [inline main.nf:202]:
```suggestion
                $clean_flag\
                ${params.readout ? "$params.readout" : ""}\
```

**Review by AlexVCaron — COMMENTED** [review]
Last modif then LGTM !

**Pull request merged**